etc/benchmarks/avro-serialization-implementations/scripts/decode/node-avro-io.js

```diff
@@ -5,8 +5,8 @@
 'use strict';
 
 var io = require('node-avro-io'),
-    avsc = require('../../../../lib');
-
+    avsc = require('../../../../lib'),
+    {Buffer} = require('buffer');
```

Comment:
This syntax isn't compatible with older Node versions, could you swap to one that is? For example:

```js
    // ...
    buffer = require('buffer');

var Buffer = buffer.Buffer;
```

Also, feel free to skip the files in `etc/` as they are not included in the distributed package.

Reply:
Yep no worries.  Yeah I think that syntax is Node 16+.